app/scripts/lib/rpc-method-middleware/handlers/eth-accounts.js

```diff
@@ -21,8 +21,8 @@ export default requestEthereumAccounts;
 
 /**
  *
- * @param {import('json-rpc-engine').JsonRpcRequest<unknown>} _req - The JSON-RPC request object.
- * @param {import('json-rpc-engine').JsonRpcResponse<true>} res - The JSON-RPC response object.
+ * @param {import('@metamask/json-rpc-engine').JsonRpcRequest<unknown>} _req - The JSON-RPC request object.
```

Comment:
```suggestion
 * @param {import('@metamask/json-rpc-engine').JsonRpcRequest} _req - The JSON-RPC request object.
```

If we want to be verbose, this is also a possibility:

```suggestion
 * @param {import('@metamask/json-rpc-engine').JsonRpcRequest<import('@metamask/utils').JsonRpcParams>} _req - The JSON-RPC request object.
```